.github/workflows/backend-integration-test.yml

```diff
@@ -25,6 +25,7 @@ jobs:
               - api/**
               - bin/**
               - catalogs/**
+              - clients/cli/**
```

Comment:
also need to add this to `.github/workflows/build.yml`

Reply:
Done